src/main/java/XlsxExcelFile.java

```diff
@@ -128,16 +150,19 @@ public XSSFRow getRow(int rowIdx) {
     @Override
     public int findEmptyCellColumnAtFixedRow(int rowIdx, int startColIdx) {
         int colIdx = startColIdx;
-        XSSFCell content;
+        XSSFCell cell;
+        CellContent content;
+        XSSFRow row = getRow(rowIdx);
+        if(row == null)
+        	return colIdx;
+        colIdx--;
```

Comment:
same as above. Moreover, we also could export the definition of this method to the abstract class `ExcelFile`